onecgiar-pr-client/src/app/pages/type-one-report/pages/tor-key-result-story/tor-key-result-story.component.ts

```diff
@@ -18,59 +21,51 @@ export class TorKeyResultStoryComponent {
   GET_keyResultStoryInitiativeId() {
     this.api.resultsSE.GET_keyResultStoryInitiativeId(this.typeOneReportSE.getInitiativeID(this.typeOneReportSE.initiativeSelected)?.id, this.typeOneReportSE.phaseSelected).subscribe(({ response }) => {
       this.typeOneReportSE.keyResultStoryData = response;
-      console.log(response);
-      //(response);
       this.tablesList = [];
       response.forEach(table => {
+        console.log(table)
```

Comment:
@yecksin is this necessary?

Reply:
Yes, it isn't necessary. I'm going to delete it.